kolena/dataset/_common.py

```diff
@@ -65,6 +65,16 @@ def _validate_dataframe_ids_uniqueness(df: pd.DataFrame, id_fields: List[str]) -
         raise InputValidationError("id fields must be hashable") from e
 
 
+def validate_dataframe_not_empty(df: pd.DataFrame) -> None:
+    if df.empty:
+        raise InputValidationError("dataframe is empty")
+
+
+def validate_name_not_empty(name: str) -> None:
+    if not name.replace(" ", ""):
```

Comment:
i think we should use [`strip`](https://docs.python.org/3.10/library/stdtypes.html#str.strip) at here

Reply:
or we can add the normalization and validation in the sdk-server as well

Reply:
True, `strip()` is simpler - changing in a sec. I'm gonna make a server side change too but wanted to change client side first